velox/exec/tests/JoinFuzzer.cpp

```diff
@@ -1017,9 +1016,16 @@ void JoinFuzzer::verify(core::JoinType joinType) {
       LOG(INFO) << "Testing plan #" << i << " with spilling";
       actual = execute(altPlans[i], /*=injectSpill=*/true);
       if (actual != nullptr && expected != nullptr) {
-        VELOX_CHECK(
-            assertEqualResults({expected}, {actual}),
-            "Logically equivalent plans produced different results");
+        try {
+          VELOX_CHECK(
+              assertEqualResults({expected}, {actual}),
+              "Logically equivalent plans produced different results");
+        } catch (const VeloxException& e) {
+          LOG(ERROR) << "Expected\n"
+                     << expected->toString(0, expected->size()) << "\nActual\n"
+                     << actual->toString(0, actual->size());
```

Comment:
The expected and actual result vectors may be large. What about we use VLOG(1) instead of LOG(ERROR) to avoid always making the log too long? (Logging through VLOG(1) can be turned on by passing the `-v=1` flag to the fuzzer test command.)

Reply:
I guess we will only log once on the first failure? So it should be ok? Thanks!